testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -590,7 +590,7 @@ Feature: Synchronize products in the products page of the Setup Wizard
 @rocky8_minion
   Scenario: Add Rocky Linux 8
     When I use spacewalk-common-channel to add channel "rockylinux8 rockylinux8-appstream rockylinux8-extras rockylinux8-uyuni-client-devel" with arch "x86_64"
-    And I wait until all synchronized channels for "res8" have finished
+    And I wait until all synchronized channels for "rockylinux-8" have finished
```

Comment:
I'm just fine for now with this change, so we fix it now.
But... it's weird that in Uyuni we cover Rocky 8 with Pure Rocky 8 channels, while in SUMA we cover Rocky 8 with Liberty 8 channels.

Reply:
I was assuming it was intentional to do so ??